libskale/SnapshotHashAgent.cpp

```diff
@@ -301,7 +301,7 @@ std::vector< std::string > SnapshotHashAgent::getNodesToDownloadSnapshotFrom(
                                               .convert_to< std::string >();
 
                     auto snapshotData = askNodeForHash( nodeUrl, blockNumber );
```

Comment:
Suggest to replace `get<i>` and work with tuples like this to improve code readability:
```
auto [hash, signature, publicKey] = snapshotData;
if ( hash != dev::h256() )
```